ansible/roles/cyhy_commander/tasks/main.yml

```diff
@@ -98,3 +98,91 @@
   file:
     path: /tmp/mongo_users.yml
     state: absent
+
+#
+# Grab the files we need to create the "places" collection from cyhy-core
+#
+- name: Grab GitHub OAuth token from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: github_oauth_token.yml
+    dest: /tmp/github_oauth_token.yml
+    mode: get
+  become: no
+
+- name: Load GitHub OAuth token from YML file
+  local_action:
+    module: include_vars
+    file: /tmp/github_oauth_token.yml
+  become: no
+
+- name: Create the /tmp/cyhy-places/scripts directory
+  file:
+    path: /tmp/cyhy-places/scripts
+    state: directory
+
+- name: Create the /tmp/cyhy-places/extras directory
+  file:
+    path: /tmp/cyhy-places/extras
+    state: directory
+
+- name: Download load_places.sh from GitHub
+  shell: "curl -o /tmp/cyhy-places/scripts/load_places.sh https://{{ github_oauth_token }}@raw.githubusercontent.com/jsf9k/cyhy-core/develop/var/load_places.sh"
```

Comment:
You could use the Ansible module [`get_url`](https://docs.ansible.com/ansible/latest/modules/get_url_module.html) here and for the two similar lines below.  I think that's a little cleaner than `shell`.